### ISSUE TITLE
redis.exceptions.BusyLoadingError: Redis is loading the dataset in memory 

### ISSUE THREAD

**Issue by faustus25** [MISP/misp-dashboard#74]
Redis errors once you execute "start_all.sh".

Can you add some mechanism (idle time) to handle to this exception until the data loading has completed?

````
Traceback (most recent call last):
  File "/usr/local/src/misp-dashboard/DASHENV/lib/python3.5/site-packages/redis/client.py", line 668, in execute_command
    return self.parse_response(connection, command_name, **options)
  File "/usr/local/src/misp-dashboard/DASHENV/lib/python3.5/site-packages/redis/client.py", line 680, in parse_response
    response = connection.read_response()
  File "/usr/local/src/misp-dashboard/DASHENV/lib/python3.5/site-packages/redis/connection.py", line 624, in read_response
    response = self._parser.read_response()
  File "/usr/local/src/misp-dashboard/DASHENV/lib/python3.5/site-packages/redis/connection.py", line 301, in read_response
    raise error
redis.exceptions.BusyLoadingError: Redis is loading the dataset in memory

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./zmq_dispatcher.py", line 299, in <module>
    main(args.sleeptime)
  File "./zmq_dispatcher.py", line 264, in main
    content = serv_list.rpop(LISTNAME)
  File "/usr/local/src/misp-dashboard/DASHENV/lib/python3.5/site-packages/redis/client.py", line 1377, in rpop
    return self.execute_command('RPOP', name)
  File "/usr/local/src/misp-dashboard/DASHENV/lib/python3.5/site-packages/redis/client.py", line 674, in execute_command
    return self.parse_response(connection, command_name, **options)
  File "/usr/local/src/misp-dashboard/DASHENV/lib/python3.5/site-packages/redis/client.py", line 680, in parse_response
    response = connection.read_response()
  File "/usr/local/src/misp-dashboard/DASHENV/lib/python3.5/site-packages/redis/connection.py", line 624, in read_response
    response = self._parser.read_response()
  File "/usr/local/src/misp-dashboard/DASHENV/lib/python3.5/site-packages/redis/connection.py", line 301, in read_response
    raise error
redis.exceptions.BusyLoadingError: Redis is loading the dataset in memory

````

**Comment by mokaddem** [comment]
Indeed, having something like this could be nice:
```
while 1:
    if redis_started():
        break;
```

**Comment by mokaddem** [comment]
Should be fixed with #121.
Please re-open if it's not the case.
